guides/gatherers.md

```diff
@@ -69,6 +70,45 @@ facts:
 
 For extra information refer to [trento-project/agent/../gatherers/corosyncconf_test.go](https://github.com/trento-project/agent/blob/main/internal/factsengine/gatherers/corosyncconf_test.go)
 
+### corosync-cmapctl
+
+This gatherer allows accessing information outputted by `corosync-cmapctl`. It supports all of the keys outputted by `corosync-cmapctl` to be queried here.
+
+Sample arguments
+| Name                                 | Return value  
+| ------------------------------------ | --------------------------------------
+| `runtime.config.totem.consensus`     | extracted value from the command output e.g. `36000`
+| `runtime.votequorum.two_node`        | extracted value from the command output e.g. `1`
+| `totem.transport`                    | extracted value from the command output e.g. `"udpu"`
+| `runtime.config.totem.max_messages`  | extracted value from the command output e.g. `20`
+
+Specification examples:
+
+```yaml
+facts:
+  - name: totem_consensus
```

Comment:
I find too many "almost copy/paste" examples.
Let's put something different like:
```
- totem.token
- runtime.config.totem.token
- nodelist.node.0.ring0_addr
```
At the end, the idea is not put valuable examples, not a lot of examples hehe